tests/python/contrib/test_ethosu/test_codegen.py

```diff
@@ -17,19 +17,17 @@
 # pylint: disable=invalid-name, unused-argument
 import pytest
 
-pytest.importorskip("ethosu.vela")
 import numpy as np
 import tflite.Model
 
 import tvm
 import tensorflow as tf
 from tvm import relay
-from tvm.relay.backend.contrib.ethosu import util
 from tvm.relay.op.contrib.ethosu import partition_for_ethosu
-from tests.python.relay.aot.aot_test_utils import generate_ref_data
 
-from . import relay_ir_builder
 from . import infra
+pytest.importorskip("ethosu.vela")
```

Comment:
Lets keep this at the very top so all dependencies are satisfied -- as it originally were.

Reply:
ack